commands.go

```diff
@@ -437,6 +438,27 @@ func fnLogin(ce *WrappedCommandEvent) {
 	ce.Reply("Successfully logged in as %s (UUID: %s)", ce.User.SignalUsername, ce.User.SignalID)
 }
 
+var cmdLogout = &commands.FullHandler{
+	Func: wrapCommand(fnLogout),
+	Name: "logout",
+	Help: commands.HelpMeta{
+		Section:     commands.HelpSectionAuth,
+		Description: "Unlink the bridge from your Signal account.",
```

Comment:
There shouldn't be a logout command before logging out is actually supported, but `delete-session` should probably actually delete the session, not sure why it wasn't made to do that in the first place

Reply:
Maybe, but the phone sync is best effort. I figured why not have a command that retains this data if the user's intent is to immediately reconnect the same account. But if you don't want this command to exist I don't feel strongly about it, a user can achieve the same behaviour by deleting the device in the Signal app (the bridge will wipe encryption keys and keep everything else)

Reply:
So is the main purpose of the current `delete-session` to fix occasional issues with encryption keys, and to typically be followed up immediately with `login`?

In any case, would it be appropriate for `(*User).Logout` to also delete the encryption session like `delete-session` does?

Reply:
Expressed differently: **when would a user want to log out of the bridge without deleting their Signal encryption session?**  Intuitively, I'd expect a bridge logout to also delete the session (but now I understand why `delete-session` should _not_ also trigger a logout, for [the reasons you explained](https://github.com/mautrix/signal/pull/439#discussion_r1480031569)).

Reply:
Yup, you're correct about `delete-session`. The main reason I didn't make logout actually logout is because I didn't need it to at the time. I was implementing the provisioning API, trying to make it behave identical to the old Signal bridge (to avoid making changes to the Signal provisioning flow in Beeper), and during the provisioning flow Beeper called logout a bunch of times before logging in, and I didn't want to break the reconnection logic, so I just made it no-op.

The ideal would be to change the logout endpoint in the provisioning API to be a more obvious no-op (and eventually fix Beeper to be more sane during Signal provisioning so we don't need the no-op at all), then properly implement the logout command (delete keys and session in bridge, and delete device from Signal). But I don't currently have bandwidth to be a lot of help with this at the moment ;)